src/foam/u2/IntView.js

```diff
@@ -44,6 +44,7 @@ foam.CLASS({
       this.SUPER(p);
       this.min = p.min;
       this.max = p.max;
+      this.mode = p.mode;
```

Comment:
This doesn't make sense to me. `p` is a `Property` and afaik, `Property`-ies don't have a `mode` property.

Reply:
Right, so `this.mode` makes sense, but it's `p.mode` that doesn't make sense to me.